i18n/ar/README.md

```
@@ -0,0 +1,1233 @@
+
+<table>
+<tr>
+<th align="left" width="50%"><sub>FIG_003 · A</sub><br/><b>THE INDUSTRY SIGNAL</b></th>
+<th align="left" width="50%"><sub>FIG_003 · B</sub><br/><b>FOUNDATIONAL PAPERS COVERED</b></th>
+</tr>
+<tr>
+<td valign="top">
+
+> *"The hottest new programming language is English."*<br/>
+> — **Andrej Karpathy** ([tweet](https://x.com/karpathy/status/1617979122625712128))
+
```

markdownlint-cli2 (0.23.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)